### PR TITLE
NCSD-1862: Service Bus topic + subscription templates

### PR DESCRIPTION
**Pull request by rivoric**
Service Bus topic and topic subscription templates added with simple pester test

### CODE REVIEW

File: ArmTemplates/ServiceBus/servicebus-topic.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+            }
+        },
+        "defaultTTL": {
+            "type": "string",
+            "defaultValue": "P90D",
+            "metadata": {
+                "description": "Default time to live (defaults to 90 days)"
+            }
+        },
+        "maxSize": {
```
Comment (thelious1974) [inline ArmTemplates/ServiceBus/servicebus-topic.json]: What is this the max size for?   
The size of the topic? 

Also, the description is incorrect.

Comment (thelious1974) [reply ArmTemplates/ServiceBus/servicebus-topic.json]: Yes, it was the size of the topic. I suggest renaming the parameter to better indicate what it is - maybe "TopicMaxSizeInMb" ?

File: ArmTemplates/ServiceBus/servicebus-topic.json [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+                "description": "Name of an existing Service Bus namespace to add the topic to"
+            }
+        },
+        "serviceBusTopicName": {
+            "type": "string",
+            "metadata": {
+                "description": "Topic name to add to Service Bus"
+            }
+        },
+        "defaultTTL": {
```
Comment (thelious1974) [inline ArmTemplates/ServiceBus/servicebus-topic.json]: Maybe rename this to `defaultMessageTtl`  to better indicate what the ttl is applying to?

File: ArmTemplates/ServiceBus/servicebus-topic.json (hunk 1 above)
Comment (rivoric) [reply ArmTemplates/ServiceBus/servicebus-topic.json]: Updated to messageMaxSizeMb

File: ArmTemplates/ServiceBus/servicebus-topic.json (hunk 2 above)
Comment (rivoric) [reply ArmTemplates/ServiceBus/servicebus-topic.json]: Changed to messageDefaultTTL in line with other change

**Pull request merged**